src/components/ProposalStatusBadge/ProposalStatusBadge.tsx

```diff
@@ -12,6 +12,14 @@ const messages = {
 
 const outcomeMapping = {
   [Outcome.InProgress]: { text: messages.open, style: styles.open },
+  [Outcome.InProgressExecutionDelay]: {
+    text: messages.open,
```

Comment:
We actually had some loose designs for these terms that didn't make a whole lot of sense, but i'd be down with maybe making this more explicit (check w/ forrest as well) -- something like:

Execution Delay

Awaiting Execution

and changing the color to orange per design for pending states here:
https://www.figma.com/file/csnO7UcYJpF1iu6t0V4F4T/Audius-Protocol-Dashboard?node-id=128%3A0